chapters/classes.tex

```diff
@@ -387,8 +387,9 @@ \subsection{Component Variability Prefixes discrete, parameter, constant}\double
   constant during transient analysis.
 \item
   A \emph{discrete-time} variable vd has a vanishing time derivative
-  (informally \lstinline!der(vd)=0!, but it is not legal to apply the \lstinline!der()! operator
-  to discrete-time variables) and can change its values only at event
+  (informally \lstinline!der(vd)=0! between events and possibly infinite derivative at event,
```

Comment:
"possibly infinite at event_s_" instead of "possibly infinite _derivative_ at event", since we're already talking about der(vd).

Reply:
I don't like the use of _infinitive derivative_ to refer to the Dirac δ distribution.  I prefer sticking with the classical kind of differentiation where the derivative is simply not defined at the discontinuities.  To avoid reference to possible discontinuities, it could be phrased as:
> A \emph{discrete-time} variable vd has a vanishing time derivative that is only defined between events (note that this is not the same as saying that der(vd) = 0 almost everywhere, as the derivative isn't even defined at the events)

Reply:
I tried to update it based on this, but in a slightly more verbose way.